.gitignore

```diff
@@ -7,6 +7,10 @@ bundles/io/org.openhab.io.multimedia.tts.marytts/lib/
 .metadata/
 */plugin.xml_gen
 targetplatform/*.launch
+.project
```

Comment:
Please remove these ignores and commit your .* files of the binding - otherwise the binding won't turn up in the standard IDE setup.